src/components/Collections/TokenDetail/index.tsx

```diff
@@ -257,118 +303,47 @@ function TokenDetail({ contractAddress, tokenId }: TokenDetailProps) {
                 </Flex>
               </Flex>
             ) : null}
-
-            <Flex
-              justify="space-between"
-              align="center"
-              w="100%"
-              px={8}
-              pb={6}
-              borderBottom="1px solid"
-              borderColor="brand.lightBlue"
+            <Heading color="brand.black" size="xl">
+              {token.title}
+            </Heading>
+            <Heading color="brand.darkGray" size="md" mt={[2, 4]}>
+              Creator: {token.metadata?.creators?.join(', ') || 'Unkown'}
```

Comment:
Change label to `Minter:`. See other comment on creator vs minter.

Reply:
Okay.